week-1/interpret/percentage-change.js

```diff
@@ -9,12 +9,32 @@ const percentageChange = (priceDifference / carPrice) * 100;
 
 console.log(`The percentage change is ${percentageChange}`);
 
+
 // Read the code and then answer the questions below
 
 // a) How many function calls are there in this file? Write down all the lines where a function call is made
 
+<<<<<<< HEAD
+// There are 3. Line 4, Line 7 and Line 10.
+
+// b) Run the code and identify the line where the error is coming from - why is this error occurring? How can you fix this problem?
+=======
 // b) Identify all the lines that are variable reassignment statements
+>>>>>>> 03fce56db903fb986ff36f73e4d57cebfa18707c
 
 // c) Identify all the lines that are variable declarations
 
+<<<<<<< HEAD
+// There are 2. On line 4 and line 5
+
+// d) Identify all the lines that are variable declarations
+
+// There are 2. On line 7 and line 8
+
+// e) Describe what the expression Number(carPrice.replaceAll(",","")) is doing - what is the purpose of this expression?
+
+// This expression is saying that I want to replace the data type of carPrice from a string to a number
```

Comment:
This is not quite correct.  The first step is the `replaceAll` function where the value is still a string but what happens is that all the commas are removed from the string leaving `"10000"`.  The second step is the `Number` function which converts the String to a number.